libs/common/src/assets/i18n/fr.json

```
@@ -569,6 +571,21 @@
 	"kb.form.invalid-slug": "Knowledge Box slug invalide",
 	"kb.form.name": "Nom de la Knowledge Box",
 	"kb.form.required-name": "Champs requis",
+	"kb.metrics.health-status.description": "Voyez si votre pipeline RAG est en bonne santé en vérifiant les valeurs moyennes, minimales et maximales de chaque indicateur sur la période sélectionnée.",
```

```suggestion
	"kb.metrics.health-status.description": "Voir si votre pipeline RAG est en bonne santé en vérifiant les valeurs moyennes, minimales et maximales de chaque indicateur sur la période sélectionnée.",
```
The infinitive sounds a bit better here, no?